app/portainer/views/sidebar/sidebarController.js

```diff
@@ -1,38 +1,39 @@
-angular.module('portainer.app').controller('SidebarController', [
-  '$q',
-  '$scope',
-  'StateManager',
-  'Notifications',
-  'Authentication',
-  'UserService',
-  function ($q, $scope, StateManager, Notifications, Authentication, UserService) {
-    function checkPermissions(memberships) {
-      var isLeader = false;
-      angular.forEach(memberships, function (membership) {
-        if (membership.Role === 1) {
-          isLeader = true;
-        }
-      });
-      $scope.isTeamLeader = isLeader;
-    }
+import angular from 'angular';
```

Comment:
I think this file can be reverted, endpoint id isn't needed on the sidebar anymore